media-gfx/exif/exif-0.6.22-r1.ebuild

```diff
@@ -22,6 +22,11 @@ DEPEND="
 "
 RDEPEND="${DEPEND}"
 
+PATCHES=(
+	"${FILESDIR}"/empty-string-check.patch
+	"${FILESDIR}"/fix-empty-string-check.patch
+)
```

Comment:
Why not combine these into one patch?

Reply:
I think I misunderstood you - I thought you meant manually combining the patches.

It's okay to have one patch file, where you link to both commits in order at the top of it, and paste one .patch then another, in the same file.

Reply:
Thakn you and my apologies for getting confused (and confusing you)!